releng/target-platform/target-platform.tpd

```diff
@@ -2,7 +2,7 @@ target "bazel-eclipse-feature-target-platform" with source configurePhase requir
 
 environment JavaSE-11
 
-location "https://download.eclipse.org/eclipse/updates/4.20-I-builds/I20210429-1800/" {
+location "https://download.eclipse.org/eclipse/updates/4.20/R-4.20-202106111600/" {
```

Comment:
@plaird The `R-` is the indicator for release build. So this is good now. I only used the I-builds initially because the release wasn't available yet.